media-sound/picard/picard-2.1.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="MusicBrainz Picard music tagger"
+DESCRIPTION="Tag your music with the next generation MusicBrainz tagger"
+HOMEPAGE="https://picard.musicbrainz.org/"
+COPYRIGHT="2004-2019 Robert Kaye, Lukas Lalinsky, Laurent Monin, Sambhav Kothari, Philipp Wolfer and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="ftp://ftp.eu.metabrainz.org/pub/musicbrainz/picard/picard-$portVersion.tar.gz"
+CHECKSUM_SHA256="8e044fe68c44d345c19f98952c3c7f20c72da74dbac9ce7c7b0621e2d69885a7"
+SOURCE_DIR="picard-release-$portVersion"
+PATCHES="picard-$portVersion.patchset"
+ADDITIONAL_FILES="picard.rdef.in"
+
+ARCHITECTURES="x86 x86_64"
+
+PROVIDES="
+		picard = $portVersion
+		cmd:picard = $portVersion
+		"
+REQUIRES="
+		haiku
+		cmd:python3.6
+		pyqt
+		mutagen_python36
+		discid_python36
+		cmd:fpcalc
+		"
+
+BUILD_REQUIRES="
+		haiku_devel
+		cmd:gcc
+		cmd:msgfmt
+		cmd:python3.6
+		setuptools_python36
+		pyqt
+		"
+
+BUILD()
+{
+	LC_ALL=en_US.UTF-8
+	python3.6 setup.py config
+}
+
+INSTALL()
+{
+	LC_ALL=en_US.UTF-8
+	python3.6 setup.py install \
+		--root=/ --prefix=$prefix \
+		--install-data=$dataDir \
+		--install-locales=$dataDir/share/locale/ \
+		--disable-autoupdate
+
+	mkdir -p $appsDir
+	mv $binDir/picard $appsDir/Picard
+	rm -rf $binDir
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3 | cut -d~ -f1`"
+
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/picard.rdef.in > $sourceDir/build/picard.rdef
+
+	rc $sourceDir/build/picard.rdef
+	resattr -o $appsDir/Picard $sourceDir/build/picard.rsrc
```

Comment:
These 2 lines can be replaced with `addResourcesToBinaries  $sourceDir/build/picard.rdef`

Reply:
Thanks, much simpler. Done.

Reply:
I reverted this in caf4b343e5c7b3cc794fef15e7121448fae26660 . I don't think addResourcesToBinaries works here, the executable is a Python script. addResourcesToBinaries destroyed it and turned it into binary garbage.

I took the `rc` + `resattr` combo from another recipe which also had a script based launcher. I don't fully understand the background yet how this is handled on Haiku, but it seems using `resattr` the details are stored as file system attributes instead of embedding them into the binary file. If there is a better way to handle that, let me know.

Reply:
Ah, yes. addResourcesToBinaries on works with, well, binaries. So `rc` + `resattr` combo is good enough.